src/components/DrawerContent.js

```diff
@@ -1,38 +1,91 @@
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, View, TouchableOpacity, Text, Image } from 'react-native';
 import React from 'react';
 import { connect } from 'react-redux';
-import { baseSelectors } from '../store/base/ducks';
+import { baseSelectors, baseActions } from '../store/base/ducks';
 
 import gStyle from '../constants/gStyle';
 import colors from '../constants/colors';
 
 const styles = StyleSheet.create({
   drawerContentContainer: {
-    alignItems: 'center',
+    backgroundColor: colors.purpleDark,
+    flex: 1,
+  },
+  drawerLinksContainer: {
     backgroundColor: colors.purple,
     flex: 1,
-    justifyContent: 'center',
+    paddingHorizontal: 15,
+  },
+  baseLinkContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: 15,
   },
   text: {
```

Comment:
lets start refactoring the styles, create a new sharedStyles.js, and put all the text at the very least tehre, so we can stay consistent and stick to just a few styles